tests/functional/ui/test_health_overview.py

```
@@ -612,6 +612,13 @@ def test_health_score_changes_based_on_alert_severity(
             self.restart_pod()
             logger.info("Waiting for 120 sec for pod to restart")
             time.sleep(120)
+            api = PrometheusAPI(threading_lock=threading_lock)
+            api.refresh_connection()
+            alerts = api.wait_for_alert(name=alert_name, state="firing", sleep=60)
+            assert (
+                len(alerts) > 0
+            ), f"Alert {alert_name} is not in firing state after unsilencing"
+            logger.info(f"Alert {alert_name} confirmed in firing state")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)